### DISCUSSION TITLE
It doesn't work with the Burmese language characters

### DISCUSSION THREAD

**Discussion by tkoop** [spatie/browsershot#848] [Bugs]
**Concerning this code:**

```php
Browsershot::html("<h1>Title: မြန်မာ</h1><p>Words: ဒါက အလုပ်ဖြစ်မယ် မထင်ဘူး။</p>")->noSandbox()->save("my.pdf");
```

**Result:**
I expect the PDF to contain these Burmese characters, but it doesn't. They are blank, but the English words are there.
![image](https://github.com/spatie/browsershot/assets/294852/c349f1e9-7a76-496c-bdaf-60e7ee2292bd)

**Actual print from Chrome**
When I load this HTML into Chrome and print to pdf, this is the PDF it creates: (This is the expected behavior.)
![image](https://github.com/spatie/browsershot/assets/294852/a1425e66-0ce0-4b4a-9417-f4f255def573)

If I encode the unicode characters like this, it does the same thing. It doesn't fix it.
```
<h1>Title: &#4121;&#4156;&#4116;&#4154;&#4121;&#4140;</h1><p>Words: &#4114;&#4139;&#4096; &#4129;&#4124;&#4143;&#4117;&#4154;&#4118;&#4156;&#4101;&#4154;&#4121;&#4122;&#4154; &#4121;&#4113;&#4100;&#4154;&#4120;&#4144;&#4152;&#4171;</p>  
```

**Comment by tkoop** [comment]
This has been solved, and this is how I solved it. It was a missing font. I was running browsershot in Docker (that didn't have the required font), and I was running Chrome manually on my system that did have the correct font installed.

To fix:
```bash
sudo apt-get install fonts-noto
```

**Reply by ThantHtetAung101** [reply]
Can you share your browsershot code for that? I've been trying to use this for weeks but I can't even get it to work.
Error: ENOENT: no such file or directory, mkdtemp 'undefined\\temp\\puppeteer_dev_chrome_profile-XXXXXX'

My code 
`Browsershot::html('<h1>Hello World</h1>')
            ->setNodeBinary('C:\Program Files\nodejs\node')
            ->setNpmBinary('C:\Program Files\nodejs\npm')
            ->setChromePath('C:\Program Files\Google\Chrome\Application\chrome')
            ->save('test.jpg');
`

Does browershot only work on linux or mac?
